### DISCUSSION TITLE
DaVinci Resolve PostgreSQL Workflow Tools

### DISCUSSION THREAD

**Discussion by sethgoldin** [sethgoldin/davinci-resolve-postgresql-workflow-tools#36] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/sethgoldin/davinci-resolve-postgresql-workflow-tools/releases/tag/v2.0'>DaVinci Resolve PostgreSQL Workflow Tools</a>.</em>

**Comment by ayeyo0** [comment]
Hello Stupid question but by optimize what exactly do you mean? im guessing that its just how often it will delete older backups. but just wanted to confirm

**Reply by sethgoldin** [reply]
In DaVinci Resolve, you'll notice that there's an `Optimize` button for PostgreSQL databases. That's Blackmagic Design's shorthand for [vacuumdb](https://www.postgresql.org/docs/13/app-vacuumdb.html) and [reindexdb](https://www.postgresql.org/docs/13/sql-reindex.html).

These tools do _not_ delete older backups.

**Comment by ayeyo0** [comment]
Understood I see that now. One additional question with this tool am I able to specify what time to start the backups and optimization?
